base/src/main/resources/db/migration/V83__change_sys_tenant_to_short.sql

```diff
@@ -0,0 +1,218 @@
+/*
+ *  Copyright (c) 2021 Vitasystems GmbH.
+ *
+ *  This file is part of project EHRbase
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *   software distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and  limitations under the License.
+ *
+ */
+
+-- prepare id column
+CREATE SEQUENCE ehr.sys_tenant_id_seq AS SMALLINT;
+
+CREATE OR REPLACE FUNCTION ehr.next() RETURNS SMALLINT
```

Comment:
ehr.next() runs with the person who executed the script, but select ehr.next(); can be called by the restricted user

Reply:
@HolgerReiseVSys this is needed do to the having 2 users. but maybe  next() should be renamed to next_sys_tenant_id_seq()

Reply:
renamed

Reply:
We could also give the restricted user rights to all new sequences like we did for table but that would again be a braking change. since this has to be run as postgres.